providers/src/airflow/providers/openlineage/provider.yaml

```
@@ -51,7 +51,7 @@ versions:
 
 dependencies:
   - apache-airflow>=2.8.0
-  - apache-airflow-providers-common-sql>=1.6.0
+  - apache-airflow-providers-common-sql>=1.17.0
```

```suggestion
  - apache-airflow-providers-common-sql>=1.20.0
```

We need to bump the versions again

---

Hopefully, this is done.